js/tiles.js

```diff
@@ -991,6 +991,14 @@ const door = {
             this.ground.show();
         }
     }),
+    blue: Object.assign({}, OpenDoors, {
+        createImages: function() {
+            this.wallTop = this.createImage("tiles/rooms/wall/blueWall.svg");
```

Comment:
You show a wall there but you use OpenDoors as a base. This means there is a wall but shown but one can walk through it. Would you like to add information about entering and leaving on the top as `false`?